packages/accordions/src/styled/timeline/StyledItem.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { StyledSeparator } from './StyledSeparator';
+import { StyledTimelineContent } from './StyledContent';
+import { StyledOppositeContent } from './StyledOppositeContent';
+
+const COMPONENT_ID = 'timeline.item';
+
+interface IStyledTimelineItem {
+  surfaceColor?: string;
```

Comment:
Is this right name for this prop would something more obvious like `iconBackgroundColor` be more obvious? Feel free to ignore, I just haven't seen the `surface` wording used anywhere else in garden.